nisystemlink/clients/feeds/models/_feed.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+from ._platform import Platform
+
+
+class Feed(JsonModel):
+    """Feed model."""
+
+    id: Optional[str] = None
+    """The auto-generated ID of the feed."""
+    name: Optional[str] = None
+    """The name of the feed."""
+    description: Optional[str] = None
+    """The description of the feed."""
+    platform: Platform
+    """The platform of the feed, the following package extensions are available: .nipkg for
+    windows feeds, .ipk and .deb for ni-linux-rt feeds.
+    """
+    workspace: Optional[str] = None
+    """The ID of the workspace this feed belongs to."""
+    updated_at: str
+    """The date of the latest feed update"""
+    created_at: str
```

Comment:
All fields of `Feed` are optional (no red *)

![image](https://github.com/user-attachments/assets/d4b4cc93-f47e-46f7-ab4b-9a19f72bcfaf)